apps/mobile/src/app/wallet/_layout.tsx

```diff
@@ -133,9 +133,13 @@ export default function StackLayout() {
         options={{ header: () => NavigationWalletSettings() }}
       />
       <Stack.Screen
-        name="wallet-settings/configure/[wallet]/[account]"
+        name="wallet-settings/configure/[wallet]/[account]/index"
```

Comment:
I created an issue to refactor these routes at some point, would love to see them nested with the other settings:
https://github.com/leather-io/issues/issues/301